### PR TITLE
Species name in examine is lowercase

### PR DESCRIPTION
**Pull request by honkpocket**
## About The Pull Request

Totally a nitpick PR.

image

## Why It's Good For The Game

Grammar.

### CODE REVIEW

**Comment by CliffracerX** [comment]
Would species names not be considered proper(-ish) nouns here?  I.E, She is a Thing of Type
~~I may just be too FromSoft boss nameplate coded with this take~~

**Comment by honkpocket** [comment]
I'm pretty sure this is correct. To prevent confusion, the only part getting lower-case'd is the true species title, not the one the player can customize.

File: code/modules/mob/living/carbon/examine.dm [hunk 1]
```diff
@@ -27,7 +27,7 @@
 		. += "[dna.features["flavor_short_desc"]] [get_extended_description_href("\[👁️\]")]"
 	ADD_NEWLINE_IF_NECESSARY(.)
 	if (dna.features["custom_species_name"])
-		. += "[t_He] [t_is] [prefix_a_or_an(dna.features["custom_species_name"])] <em>[get_species_description_href(dna.features["custom_species_name"])]</em> of [dna.species.name] physiology."
+		. += "[t_He] [t_is] [prefix_a_or_an(dna.features["custom_species_name"])] <em>[get_species_description_href(dna.features["custom_species_name"])]</em> of [LOWER_TEXT(dna.species.name)] physiology."
```
Comment (CliffracerX) [inline code/modules/mob/living/carbon/examine.dm:30]: Needs a /// DOPPLER SHIFT EDIT marker but otherwise, this is ready.

**Review by CliffracerX — APPROVED** [review]
Once the edit marker nitpick is resolved, this should be good to go.

File: code/modules/mob/living/carbon/examine.dm (hunk 1 above)
Comment (honkpocket) [reply code/modules/mob/living/carbon/examine.dm:30]: This chunk of code is already inbetween two doppler edit comments!

Comment (CliffracerX) [reply code/modules/mob/living/carbon/examine.dm:30]: ough, disregard then!

**Pull request merged**